Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN make build-linux
 FROM alpine:edge
 
 # Install ca-certificates
-RUN apk add --update ca-certificates
+RUN apk add --update ca-certificates libstdc++
```

Comment:
Is this change specific to M1? Then, it would be nice to state it in the comment or description of the PR.

Reply:
I know it is for using  BLS of `line/ostracon`.

Reply:
I tried it on M1 macbook docker linux/arm64 and linux/amd64 environment, but the latest github.com/herumi/bls-eth-go-binary seems to require libstdc++.